bears/general/IndentationBear.py

```diff
@@ -60,14 +65,13 @@ def run(self,
             return
 
         insert = ' '*tab_width if use_spaces else '\t'
+        no_indent_file = [line.lstrip() if line.lstrip() is not "" else "\n"
+                          for line_nr, line in enumerate(file)]
```

Comment:
all strings besides empty strings have a boolean value of `True`, right?

so you could write:
```python
no_indent_file = [line.lstrip() if line.lstrip() else "\n"
                          for line_nr, line in enumerate(file)]
```

I'm not always a fan of writing everything as compact as possible but here I feel like that makes it a little less convoluted.